pom.xml

```diff
@@ -215,10 +215,19 @@
         <groupId>com.diffplug.spotless</groupId>
         <artifactId>spotless-maven-plugin</artifactId>
         <version>2.12.2</version>
+        <executions>
+          <execution>
+            <phase>validate</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
```

Comment:
Was this from the README ?
I would probably add a comment here or in README how to run spotless manually then to fix it.
(I can't remember i they added it to the error output)

Reply:
The failing output includes the suggestion to run `mvn spotless:apply`.